Makefile

```diff
@@ -47,7 +47,7 @@ publish:
 
 # resolve dependencies from remote artifact registries
 deps:
-	gem install bundler
+	gem install bundler --version=2.0.1
```

Comment:
Please change bundler version lock down to 1.17.3 due to 2.x carrying Ruby >= 2.3 requirement, which is not fine for many host environments which includes older version of Ruby by default.

Note: bundler is the first gem to install in our Ruby stack, hence it needs to support the host's Ruby environment. We can upgrade to 2.x after upgrading the host after ensuring that all OSes that we support have latest versions that support Ruby 2.3 or newer by default.